src/devtools/client/themes/variables.css

```diff
@@ -548,9 +548,9 @@
   --replaying-paused-marker-stroke: var(--secondary-accent-stroke);
   --replay-head-background: var(--purple-50);
 
-  /* Gradient fade */
-  --tab-bar-gradient-opaque: rgba(0, 0, 0, 1);
-  --tab-bar-gradient-transparent: rgba(0, 0, 0, 0);
+  /* Loading screen */
+  --loading-boxes: var(--chrome);
+  --loading-background: var(--theme-base-100);
```

Comment:
Kind of interesting to see us using different variables (and the inverse for `var(--chrome)`!) for the same bits of UI between the themes.